features/old/providers/settings.feature

```diff
@@ -6,14 +7,30 @@ Feature: Settings management
   Background:
     Given a provider is logged in
 
-  @javascript
   Scenario: Strong password setting
     And I go to the usage rules settings page
     When I check "Strong passwords"
     And I press "Update Settings"
     Then I should see the settings updated
-    And provider "foo.3scale.localhost" should have strong passwords enabled
+    And the provider should have strong passwords enabled
     When I uncheck "Strong passwords"
     And I press "Update Settings"
     Then I should see the settings updated
-    And provider "foo.3scale.localhost" should have strong passwords disabled
+    And the provider should have strong passwords disabled
+
+  Scenario: Account approval required checkbox is enabled
+    Given the provider has 1 account plan
+    When I go to the usage rules settings page
+    Then field "Account approval required" is not disabled
+
+  Scenario: Account approval required checkbox is disabled
+    Given the provider has 2 account plans
+    And the provider has "Account plans UI visible" set to "true"
+    When I go to the usage rules settings page
+    Then field "Account approval required" is disabled
+
+  Scenario: Account approval required checkbox is not visible
+    Given the provider has 2 account plans
+    And the provider has "Account plans UI visible" set to "false"
+    When I go to the usage rules settings page
+    Then I should not see "Account approval required"
```

Comment:
This looks really clear now. I can learn how it works from the scenario text, which is the point of cucumbers AFAIU.